tests/forms_tests/templatetags/tags.py

```diff
@@ -10,7 +10,7 @@ def render(self, context):
         self.count += 1
         for v in context.flatten().values():
             try:
-                v.render()
+                str(v.widget)
```

Comment:
This for: 

```pytb
forms_tests/tests/test_forms.py", line 5101, in 
test_cyclic_context_boundfield_render
    self.fail("Cyclic reference in BoundField.render().")
```

@smithdc1 - you added this test 😅 — what are we catching here? (In the wild, I don't think I've **ever** called `BoundField.render()` directly, but you have it doc'd as a the "per-request" option for customising the field template.

Reply:
I see https://github.com/django/django/pull/16247#issuecomment-1314356116